pandas/tests/series/test_subclass.py

```diff
@@ -39,6 +39,9 @@ def test_subclass_unstack(self):
 
         tm.assert_frame_equal(res, exp)
 
+    def test_subclass_empty_repr(self):
```

Comment:
Reference PR number as comment below this

Reply:
General question: if a PR is already passing, is it worth another commit and CI run just for this?

I'd prefer to just merge and be done with it.

Reply:
yep that’s cool